### PR TITLE
PWN-9305 - make changes according to `add money` logic

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-9305

## Description of Work

PWN-9305 - make changes according to add money logic

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/home/ui/topup/TopUpWalletPresenter.kt [hunk 1]
```diff
@@ -54,6 +54,14 @@ class TopUpWalletPresenter(
...
 
     override fun onBankTransferClicked() {
+        if (!strigaSignupFeatureToggle.isFeatureEnabled) {
+            launch {
+                val tokenToBuy = userInteractor.getSingleTokenForBuy()
+                tokenToBuy?.let {
+                    view?.navigateToBuyWithTransfer(it)
+                }
+            }
+        }
```
Comment (eduardmaximovich) [inline app/src/main/java/org/p2p/wallet/home/ui/topup/TopUpWalletPresenter.kt:64]: I think there is missing `return` statement

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/home/ui/topup/TopUpWalletPresenter.kt:64]: Shouldn't we hide the transfer item if striga is not enabled?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/moonpay/ui/new/NewBuyPresenter.kt [hunk 2]
```diff
@@ -103,7 +104,9 @@ class NewBuyPresenter(
             view?.showLoading(isLoading = true)
             currentAlphaCode = paymentMethodsInteractor.getBankTransferAlphaCode()
 
-            val availablePaymentMethods = paymentMethodsInteractor.getAvailablePaymentMethods(currentAlphaCode)
+            val availablePaymentMethods = paymentMethodsInteractor.getAvailablePaymentMethods(
+                currentAlphaCode, preselectedMethodType
```
Comment (eduardmaximovich) [inline app/src/main/java/org/p2p/wallet/moonpay/ui/new/NewBuyPresenter.kt:108]: to make it clear, let `alphaCode` variables always represent the standard of their value:
```suggestion
                currentAlpha3Code, preselectedMethodType
```

**Review by eduardmaximovich — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/home/ui/topup/TopUpWalletPresenter.kt (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/home/ui/topup/TopUpWalletPresenter.kt:64]: no
in new logic we should not

Comment (eduardmaximovich) [reply app/src/main/java/org/p2p/wallet/home/ui/topup/TopUpWalletPresenter.kt:64]: @gslevinkov hah, yeah, it hides the item in attachView if striga is disabled and user is web3-authorized
@kabramovp2p have you checked this?

**Pull request merged**